.gitignore

```diff
@@ -11,3 +11,4 @@ __pycache__
 **config.py
 **database.db
 **send.bat
+errorlog.txt
```

Comment:
does this need to be `**errorlog.txt` to make sure it catches 'error.txt' in every folder?